Plots/Contours/NCL_lb_2.py

```diff
@@ -0,0 +1,47 @@
+"""
+lb_2
+===============
+Plots/Contours/Lines
+"""
+
+###############################################################################
+# 
+# import modules
+import numpy as np
+import xarray as xr
+import cartopy.feature as cfeature
+import cartopy.crs as ccrs
+import matplotlib.pyplot as plt
+
+
+###############################################################################
+# 
+# open data file and extract variables
+ds = xr.open_dataset('../../data/netcdf_files/atmos.nc', decode_times=False)
+v = ds.V.isel(time=0, lev = 3)
+
+###############################################################################
+# 
+# create plot
+fig = plt.figure(figsize=(10,10))
+
+ax = plt.axes(projection=ccrs.PlateCarree())
+ax.coastlines(linewidths=0.5)
+
+xticks = [-180, -150, -120, -90, -60, -30, 0, 30, 60, 90, 120, 150, 180]
```

Comment:
Rather than explicitly setting the axis labels can you automate them by using the Cartopy lat-lon formatter? See the add_lat_lon_ticklabels and nclize_axis functions in Mike's PR #34

Reply:
I tried this and it doesn't look the way I like. There are now negative East tick marks, instead of positive West tick marks. I could play around with it more. Is it okay to have different methods used throughout the gallery.
![image](https://user-images.githubusercontent.com/46687291/73090029-c0dcd000-3e94-11ea-8173-e378a7048822.png)

Reply:
Sure. It's ok to have different methods. How about adding a comment about why explicit labeling is preferable in this case?

Reply:
Can do!